src/shapes/text.class.js

```diff
@@ -1153,7 +1148,14 @@
      * @returns {Object} this
      */
     setSubscript: function(line, char) {
-      return this.subscript.apply(this, line, char);
+      return this._setScript(line, char, this.subscript);
+    },
+
+    _setScript: function(line, char, schema) {
+      var fontSize = this.getValueOfPropertyAt(line, char, 'fontSize'),
+          baseline = this.getValueOfPropertyAt(line, char, 'deltaY') + fontSize * schema.baseline;
```

Comment:
that's supposed to be ... + fontSize * -schema.baseline (follows the baseline-shift attribute from SVG)